CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
 # Change Log
 
+## Unreleased
+
+- Upgrade to `parse5@^v2.2.22` so that we can take advantage of new types info.
```

Comment:
This is an entirely internal change, so IMO shouldn't get a line in the change log.